scripts/rails_console/deduplicate_question_uids.rb

```diff
@@ -0,0 +1,397 @@
+ {:uid=>"-Jzw0qjQwbyM2twtkmWF", :count=>2071, :new_uid=>"4be830d6-dcaa-4fad-878e-f558fdcde6ab"},
+ {:uid=>"-Jzw0qjQwbyM2twtkmWG", :count=>7050, :new_uid=>"50abf821-95cb-4af9-941e-b2db98946433"},
+ {:uid=>"-Jzw0qjQwbyM2twtkmWH", :count=>2711, :new_uid=>"bc546b98-6e7e-48a9-8a61-172b1e16ac18"},
+ {:uid=>"-Jzw0qjQwbyM2twtkmWK", :count=>14617, :new_uid=>"6045a79a-9d3f-4d9c-9d96-46ac5f4f4a00"},
+ {:uid=>"-Jzw0qjQwbyM2twtkmWl", :count=>14084, :new_uid=>"9162ad57-9ff1-4fbb-b8b8-ff48bdf245bf"},
+ {:uid=>"-Jzw0qjQwbyM2twtkmWn", :count=>9588, :new_uid=>"da695b69-66cb-4857-bf9b-6d352a03b046"},
+ {:uid=>"-Jzw0qjQwbyM2twtkmWp", :count=>7021, :new_uid=>"7800ddac-a0d7-49f3-8d07-492b7442b323"},
+ {:uid=>"-Jzw0qjQwbyM2twtkmWq", :count=>10621, :new_uid=>"e845292f-7304-4369-9253-5c95270ae482"},
+ {:uid=>"-Jzw0qjQwbyM2twtkmWs", :count=>8693, :new_uid=>"807821c2-19c0-4272-ab50-6722c86a6f99"},
+ {:uid=>"-Jzw0qjQwbyM2twtkmWu", :count=>10032, :new_uid=>"22591c57-d927-46c6-a28a-35b7e97e7a7b"},
+ {:uid=>"-Jzw0qjQwbyM2twtkmWw", :count=>6131, :new_uid=>"9071a5f4-ca5c-4c7b-bc7a-20f187991f77"},
+ {:uid=>"-Jzw0qjQwbyM2twtkmWZ", :count=>3460, :new_uid=>"74448d32-1aa1-4ead-bb05-b2fa4d95207f"}]
+=end
+
+# took 13 minutes to run this in staging
+start_time = Time.now
+lower_count_questions.each do |question|
+  Response.where(question_uid: question[:uid]).find_in_batches(batch_size: 100) do |batch|
+    begin
+      responses_for_insertion = []
+      batch.each do |response|
+        duplicated_response = response.attributes.slice('author', 'text', 'feedback', 'count', 'first_attempt_count', 'child_count', 'optimal', 'weak', 'concept_results', 'spelling_error')
+
+        duplicated_response['question_uid'] = question[:new_uid]
+
+        # set created and updated at to Time.now because insert_all doesn't autopopulate
+        duplicated_response['created_at'] = Time.now
+        duplicated_response['updated_at'] = Time.now
+
+        responses_for_insertion.push(duplicated_response.symbolize_keys)
+      end
+      Response.insert_all!(responses_for_insertion)
+    rescue => e
+      puts "Failed to process response batch for question UID #{question[:uid]}: #{e.message}"
+      next
+    end
+  end
+  puts 'Processed question uid', question[:uid]
+end; puts Time.now - start_time
+
+UpdateElasticsearchWorker.perform_async(start_time.to_s, Time.now.to_s)
+
+# RUN THIS IN THE LMS
+
+# PASTE IN LOWER COUNT QUESTIONS
+
+lower_count_questions.each do |lower_count_question|
+  begin
+    new_uid = lower_count_question[:new_uid]
+
+    old_question = Question.find_by_uid(lower_count_question[:uid])
+
+    duplicated_question = old_question.dup
+
+    # update the UID and the data->key with the new UID
+    duplicated_question.uid = new_uid
+    duplicated_question.data['key'] = new_uid
+
+    # set created and updated at to nil so they get autopopulated correctly
+    duplicated_question.created_at = nil
```

Comment:
Again, I wonder if we should keep the original `created_at`, but I'm okay with using auto-values here.  It probably doesn't matter much either way.